tests/config.yaml

```diff
@@ -0,0 +1,138 @@
+hercules:
+  providers:
+    -
+      type: "flux"
+      name: "parallel"
+      cores per node: 80
+      nodes per block: 3
+      partition: "hercules"
+      account: "gsd-hpcs"
+    -
+      type: "htex"
+      name: "service"
+      cores per node: 1
+      nodes per block: 1
+      partition: "service"
+      account: "gsd-hpcs"
+    -
+      type: "htex"
+      name: "serial"
+      cores per node: 80
+      nodes per block: 1
+      partition: "hercules"
+      account: "gsd-hpcs"
+  
+  environment:
+    - "module purge"
+    - "module use /work/noaa/epic/role-epic/spack-stack/hercules/modulefiles"
+    - "module load ecflow/5.8.4"
+    - "module load mysql/8.0.31"
+    - "module use /work/noaa/epic/role-epic/spack-stack/hercules/spack-stack-1.6.0/envs/unified-env/install/modulefiles/Core"
+    - "module load stack-intel/2021.9.0"
+    - "module load stack-intel-oneapi-mpi/2021.9.0"
+    - "module load stack-python/3.10.13"
+    - "module load jedi-fv3-env"
+    - "module load ewok-env"
+    - "module load soca-env"
+    - "unset I_MPI_PMI_LIBRARY"
+    - "export CHILTEPIN_MPIF90=$MPIF90"
```

Comment:
Wondering if we've tried to pair this down? When running jedi in a pair programming session with Paul I found i needed less stuff on uwtools side to run the jedi executable. One thing I changed on my run with specificlaly the `ewok` and `soca` module loads. If nothing changes that's fine to. Thought I'd ask.